pkgs/development/compilers/rust/patches/0001-Disable-fragile-tests-libstd-net-tcp-on-Darwin-Linux.patch

```diff
@@ -32,22 +20,19 @@ index 8d1e7882e5..2c223fb315 100644
      #[test]
      fn timeouts() {
          let addr = next_test_ip4();
-@@ -1500,6 +1504,7 @@ mod tests {
+@@ -1610,6 +1614,7 @@
          assert_eq!(false, t!(stream.nodelay()));
      }
  
 +    #[cfg_attr(target_os = "macos", ignore)]
      #[test]
      fn ttl() {
          let ttl = 100;
-@@ -1568,6 +1573,7 @@ mod tests {
+@@ -1678,6 +1683,7 @@
          })
      }
  
-+    #[cfg_attr(target_os = "linux", ignore)]
++    #[cfg_attr(target_os = "macos", ignore)]
```

Comment:
This one should stay `linux` unless something has changed.

Reply:
Oh, my bad